os/sdk-modifying-coreos.md

```diff
@@ -61,57 +61,6 @@ cork enter
 
 To use the SDK chroot in the future, run `cork enter` from the above directory.
 
-### Option 2: using repo and cros\_sdk
-
-#### Install Repo
-
-The `repo` utility helps to manage the collection of git repositories that makes up Container Linux. 
-
-For newer Debian, Ubuntu, and other Debian based systems, install the repo package from your distro:
-
-    sudo apt-get install repo
-
-For systems without a packaged repo download it and add it to `$PATH`:
-
-```sh
-mkdir ~/bin
-export PATH="$PATH:$HOME/bin"
-curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo
-chmod a+x ~/bin/repo
-```
-
-You may want to add the `PATH` export to `.bashrc` or `/etc/profile.d/` so that you don’t need to set `$PATH` in every new shell.
-
-#### Bootstrap the SDK chroot
-
-Create a project directory. This will hold all of your git repos and the SDK chroot. A few gigabytes of space will be necessary.
```

Comment:
This information is missing from the option 1 text; it'd be good to move it over.